lib/pure/asyncnet.nim

```diff
@@ -650,6 +650,11 @@ proc bindAddr*(socket: AsyncSocket, port = Port(0), address = "") {.
     raiseOSError(osLastError())
   freeaddrinfo(aiList)
 
+proc hasDataBuffered*(s: AsyncSocket): bool {.since: (1, 5).} =
+  ## Determines whether an AsyncSocket has data buffered.
```

Comment:
```suggestion
  ## Determines whether an `AsyncSocket` has data buffered.
```

Reply:
or even:
```nim
proc hasDataBuffered*(self: AsyncSocket): bool {.since: (1, 5).} =
  ## Determines whether `self` has data buffered.`
```
to avoid being redundant, and using self instead of s (see nep1)